frontend/pulse3d/src/components/layouts/DashboardLayout.js

```diff
@@ -74,7 +74,13 @@ export default function DashboardLayout({ children }) {
       value={{ uploads, setUploads, setFetchUploads, pulse3dVersions, metaPulse3dVersions }}
     >
       <Container>
-        <ControlPanel />
+        <ControlPanel
+          clearTimers={() => {
```

Comment:
I don't think you need this check here, the same one is done when `ControlPanel` actually tries to to call the function